app/src/main/res/layout/dashboard_list_item_tools.xml

```diff
@@ -0,0 +1,125 @@
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:layout_margin="8dp"
+            android:elevation="@dimen/card_elevation"
+            android:onClick="@{() ->callbacks.onToolInfo(tool.code)}"
+            app:cardCornerRadius="@dimen/card_corner_radius">
+
+            <androidx.constraintlayout.widget.ConstraintLayout
+                android:layout_width="match_parent"
+                android:layout_height="match_parent">
+
+                <org.cru.godtools.base.ui.view.DaggerPicassoImageView
+                    android:layout_width="match_parent"
+                    android:layout_height="0dp"
+                    android:scaleType="centerCrop"
+                    android:src="@{banner}"
+                    app:layout_constraintBottom_toTopOf="@+id/guideline2"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toTopOf="parent"
+                    tools:srcCompat="@tools:sample/backgrounds/scenic" />
+
+                <ProgressBar
+                    style="@style/Widget.GodTools.Tools.Download.ProgressBar"
+                    android:layout_width="match_parent"
+                    android:layout_height="10dp"
+                    android:progress="@{downloadProgress}"
+                    android:visibility="gone"
+                    app:layout_constraintBottom_toTopOf="@id/guideline2"
+                    tools:max="12"
+                    tools:progress="5"
+                    tools:visibility="visible" />
+
+                <com.google.android.material.imageview.ShapeableImageView
+                    android:layout_width="28dp"
+                    android:layout_height="28dp"
+                    android:layout_marginTop="11dp"
+                    android:layout_marginEnd="11dp"
+                    android:background="@color/white"
+                    android:elevation="@dimen/card_elevation"
+                    android:onClick="@{() -> tool.isAdded ? callbacks.removeTool(tool, primaryTranslation ?? parallelTranslation) : callbacks.addTool(tool.code)}"
+                    android:tint="@color/gt_blue"
+                    app:contentPadding="3dp"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintTop_toTopOf="parent"
+                    app:selected="@{tool.isAdded}"
+                    app:shapeAppearance="@style/ShapeAppearance.GodTools.Tools.Action.Favorite"
+                    app:srcCompat="@drawable/ic_favorite" />
+
+                <androidx.constraintlayout.widget.Guideline
+                    android:id="@+id/guideline2"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:orientation="horizontal"
+                    app:layout_constraintGuide_end="62dp" />
```

Comment:
Where did you get the height of `62dp` from?

Reply:
![image](https://user-images.githubusercontent.com/970048/154098923-bcb7b61f-32d6-4362-832b-798be26b67b0.png)

Reply:
ah, I would have done it the opposite way, set a fixed height for the banner, then let the white area size automatically based on the content within it. I wouldn't have set a fixed overall height of each item, I would have done `wrap_content`. that would allow the size to scale up based on the user's accessibility settings.

Reply:
the cards aren't even the right fixed height currently, you have the `FrameLayout` set to `160dp`, with padding of `5dp`. then on the `CardView you have a margin of `8dp`. this yields the following height:
```
160dp
- 5dp x2
- 8dp x2
--------
134dp
```

134dp is less than the 150dp in figma. part of the redesign focus was to try and give a larger banner image on each tool.

Reply:
yeah, I fixed it.